### PR TITLE
Adds Const API

### PR DESCRIPTION
**Pull request by rstegg**

### CODE REVIEW

File: docs/crocks/Const.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+`Const` exposes the following constructor and instance functions:
+
+| Constructor | Instance |
+|:---|:---|
+| -- | [`inspect`](#inspect), [`value`](#value), [`type`](#type), [`equals`](#equals), [`concat`](#concat), [`map`](#map), [`ap`](#ap), [`chain`](#chain) |
+
+## Constructors
+
+--
+
```
Comment (evilsoft) [inline docs/crocks/Const.md]: Need to add a `type` function to the constructor

Comment (rstegg) [inline docs/crocks/Const.md]: Is this new?

Comment (evilsoft) [inline docs/crocks/Const.md]: Nope. Just missed it on the past reviews. They have always been there

**Comment by evilsoft** [comment]
To avoid potential confusion, I think we should keep the headings in the columns the same, but change the section headings to read:

"Available on the Constructor"
"Available on the Instance"

**Pull request merged**